src/snowflake/connector/connection_diagnostic.py

```diff
@@ -212,6 +224,10 @@ def __test_socket_get_cert(
                     conn.recv(4096).decode("utf-8")
 
                 context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+                # For some reason SSL checks fail now
+                # the test does not care if cert is valid
+                context.check_hostname = False
+                context.verify_mode = ssl.CERT_NONE
```

Comment:
although the test doesn't care, but in production do we care? this seems dangerous.
do we know why ssl checks fail?

Reply:
We don't care because we want to get the cert whether it's valid or not.  Customers have a tendency to have SSL inspecting proxies with certs that fail.  So we want this check to succeed so we can report what's in the cert.

Reply:
I went with certifi here instead.  So this is gone.  Since this is a diag, we might want to capture that with bad certs for ssl inspection, however, I'll wait and see how it plays out in customer usage.